config/locales/feedback_form.yml

```diff
@@ -0,0 +1,7 @@
+en:
+  satisfaction_levels:
+    very_satisfied: 'Very satisfied'
+    satisfied: 'Satisfied'
+    neither_satisfied_or_dissatisfied: 'Neither satisfied or dissatisfied'
+    dissatisfied: 'Dissatisfied'
+    very_dissatisfied: 'Very dissatisfied'
```

Comment:
Sometimes I see YAML strings quoted, sometimes not. I know you need to do this if they include `:`, but that’s not the case here. Not sure what our code style is for YAML.

Reply:
I don't think we have a written style for this but most people only add quotes when needed. Less noise more readable I guess. So this is a mistake, I've removed them.